mint-auditor/src/db/schema.rs

```diff
@@ -2,6 +2,14 @@
 // Run `diesel migration run` to update (inside the mint-auditor directory)
 #![allow(missing_docs)]
 
+table! {
+    audited_burns (id) {
+        id -> Nullable<Integer>,
```

Comment:
what do nullable ids get us, compared to non-nullible burn_tx_out_ids?

Reply:
Thats a requirement for SQLite to have auto-increment keys. See the top comment in https://github.com/mobilecoinfoundation/mobilecoin/blob/master/mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

Reply:
(summary is it has to be nullable to make SQLite happy)